src/Parameters/BaseParameters.php

```diff
@@ -37,6 +37,8 @@ abstract public function getHTTPQuery();
      */
     protected function buildHTTPQuery($array)
     {
+        // Potential replacement by: http_build_query($queries, '', '&', \PHP_QUERY_RFC3986);
```

Comment:
Tested for few years on a private project, RFC3986 won't work. Instead this has been ported from JS https://github.com/mconf/api-mate/blob/master/lib/vendor/bigbluebutton-api.js#L188

Reply:
ok, remark removed.